exporter/prometheusremotewriteexporter/wal.go

```diff
@@ -72,15 +74,22 @@ type walConfig struct {
 	// https://docs.google.com/document/d/1cCcoFgjDFwU2n823tKuMvrIhzHty4UDyn0IcfUHiyyI/edit#heading=h.mlf37ibqjgov
 	// but also we are using underscores "_" instead of dashes "-".
 	Directory         string        `mapstructure:"directory"`
-	NBeforeTruncation int           `mapstructure:"n_before_truncation"`
+	BufferSize        int           `mapstructure:"buffer_size"`
 	TruncateFrequency time.Duration `mapstructure:"truncate_frequency"`
 }
 
-func (wc *walConfig) nBeforeTruncation() int {
-	if wc.NBeforeTruncation <= 0 {
-		return 300
+func (wc *walConfig) bufferSize() int {
+	if wc.BufferSize > 0 {
+		return wc.BufferSize
 	}
-	return wc.NBeforeTruncation
+	return 300
```

Comment:
Nit: define all config defaults as consts somewhere close to each other and close to the config struct declaration.

Reply:
Done, thanks!